### PR TITLE
Adds Suspicious Bomber Jacket

### PR DESCRIPTION
**Pull request by Christasmurf**
## What Does This PR Do
Adds a suspicious bomber jacket to the game, as maintenance loot and an uplink item for syndicates (both nukies and normal traitors).
The jacket provides no armour or extra storage over the normal bomber jacket, it is functionally the same, as such it is NON-CONTRABAND. It costs 1 TC in the pointless badassery section of the uplink.
Maintenance loot has the same weight as no-slips to show up in any given maintenance loot pile.

## Why It's Good For The Game
Extra drip for extra customisation. Would've come with other faction ones however I need the economy rework done before those so I can mess with vending machines again.

## Images of changes
image
image

## Testing
Boot test server
Spawn in
Make self syndicate
Buy jacket, check jacket works all angles
Switch to vox, check sprite again
Spawn maint loot piles until one shows up

## Changelog
:cl:
add: Added syndicate inspired bomber jacket as maints loot and for 1TC in the uplink
/:cl:

### CODE REVIEW

**Comment by Stokes52** [comment]
It looks cool, but I think the red needs to be brighter to add more contrast. It's very difficult to differentiate between the black and the dark-dark red.

Compare with other syndicate items, the red is usually pretty bright against the black:
SyndicateKey
SyndicateToolbox
SyndicateSuit
Syndi_Hardsuit

Now compare that to the jacket sprites:
198845668-02b3b7d3-fcdf-4cc7-819e-67a57f2aec38

I suggest using a brighter red, then it will be perfect.

**Comment by McRamon** [comment]
Chameleon clothes are armored somewhat and also they are chameleon, suits from bundles are armored, combat gloves are armored iirc. I think it would fit if this jacket was slightly armored, and there is no need to make it contraband.

**Comment by Christasmurf** [comment]
> Chameleon clothes are armored somewhat and also they are chameleon, suits from bundles are armored, combat gloves are armored iirc. I think it would fit if this jacket was slightly armored, and there is no need to make it contraband.

Armour values added, description changed to reflect addition of armour. Slightly less fire and acid protection than the chameleon suit slot item but other values are the same, price in uplink unchanged

@Balance people let me know if I need to change values of armour or TC



> It looks cool, but I think the red needs to be brighter to add more contrast. It's very difficult to differentiate between the black and the dark-dark red.
> I suggest using a brighter red, then it will be perfect.

I'm unsure on this, when doing the colours I went with a more dark tunnel vibe with it similar to these items
image
image
image

As opposed to the whole "in your face bright red" that most of the other syndicate items are. I feel like this is an issue of personal preference so if more people would prefer it brighter then I'll see about that but otherwise I'm happy as it is

**Comment by AzuleUtama** [comment]
As a continuation of what Ramon mentioned already, it might be better to base some of the values off the armored suit jacket you can normally get from some bundles. The chameleon jacket armor values are very low, since the main point of that item is for a disguise whereas the bomber jacket doesn't have that trait. Said armored jacket also allows you to hold a pistol or revolver in the suit storage slot which would be very fitting for a 'suspicious' piece of clothing:
```
/obj/item/clothing/suit/storage/lawyer/blackjacket/armored
	desc = "A snappy dress jacket, reinforced with a layer of armor protecting the torso."
	allowed = list(/obj/item/tank/internals/emergency_oxygen, /obj/item/gun/projectile/revolver, /obj/item/gun/projectile/automatic/pistol)
	body_parts_covered = UPPER_TORSO|LOWER_TORSO
	cold_protection = UPPER_TORSO|LOWER_TORSO
	min_cold_protection_temperature = ARMOR_MIN_TEMP_PROTECT
	heat_protection = UPPER_TORSO|LOWER_TORSO
	max_heat_protection_temperature = ARMOR_MAX_TEMP_PROTECT
	armor = list(MELEE = 15, BULLET = 10, LASER = 15, ENERGY = 5, BOMB = 15, BIO = 0, RAD = 0, FIRE = 35, ACID = 35)
```
Note that the jacket is still slightly worse for armor than the default security armor, but much better than the chameleon clothing.

File: code/modules/clothing/suits/miscellaneous.dm [hunk 1]
```diff
@@ -786,6 +786,20 @@
 		"Vox" = 'icons/mob/clothing/species/vox/suit.dmi'
 		)
 
+/obj/item/clothing/suit/jacket/syndicatebomber
+	name = "suspicious bomber jacket"
+	desc = "A suspicious but extremely stylish jacket."
+	icon_state = "bombersyndie"
+	item_state = "bombersyndie"
+	ignore_suitadjust = 0
```
Comment (Sirryan2002) [inline code/modules/clothing/suits/miscellaneous.dm:794]:
```suggestion
	ignore_suitadjust = FALSE
```

**Review by Sirryan2002 — APPROVED** [review]
code and sprite diffs look good

**Comment by Christasmurf** [comment]
> As a continuation of what Ramon mentioned already, it might be better to base some of the values off the armored suit jacket you can normally get from some bundles. The chameleon jacket armor values are very low, since the main point of that item is for a disguise whereas the bomber jacket doesn't have that trait. Said armored jacket also allows you to hold a pistol or revolver in the suit storage slot which would be very fitting for a 'suspicious' piece of clothing:
> 
> ```
> /obj/item/clothing/suit/storage/lawyer/blackjacket/armored
> 	desc = "A snappy dress jacket, reinforced with a layer of armor protecting the torso."
> 	allowed = list(/obj/item/tank/internals/emergency_oxygen, /obj/item/gun/projectile/revolver, /obj/item/gun/projectile/automatic/pistol)
> 	body_parts_covered = UPPER_TORSO|LOWER_TORSO
> 	cold_protection = UPPER_TORSO|LOWER_TORSO
> 	min_cold_protection_temperature = ARMOR_MIN_TEMP_PROTECT
> 	heat_protection = UPPER_TORSO|LOWER_TORSO
> 	max_heat_protection_temperature = ARMOR_MAX_TEMP_PROTECT
> 	armor = list(MELEE = 15, BULLET = 10, LASER = 15, ENERGY = 5, BOMB = 15, BIO = 0, RAD = 0, FIRE = 35, ACID = 35)
> ```
> 
> Note that the jacket is still slightly worse for armor than the default security armor, but much better than the chameleon clothing.

I originally found that and was considering using the armour values from that but I'm unsure what the values even really mean in terms of in game mechanics but it seemed too close to a normal sec vest for how little it costs. I have the problem of maybe having to increase the price depending on the armour value it provides, which is another thing I am unsure on the values of as I don't really play antag. I definitely want to wait for balance team's opinion before I start messing with the values more than what they're at currently because I have absolutely no idea on how to balance them properly

**Review by AffectedArc07 — APPROVED** [review]

**Pull request merged**